validator/rpc/server.go

```diff
@@ -270,5 +240,5 @@ func (s *Server) Stop() error {
 
 // Status returns nil or credentialError.
 func (s *Server) Status() error {
-	return s.credentialError
+	return nil
```

Comment:
does this function do anything? lol is there a reason to keep it or at least add some log?

Reply:
This is part of the `Service` interface contract. It returns an error when something is wrong with the service. Which means that currently we always treat the service as healthy. If we want to add some logic here, it would be better to do it in another PR.